src/main/src/jetty/override-web.xml

```diff
@@ -31,4 +31,10 @@
        <res-type>javax.sql.DataSource</res-type>
        <res-auth>Container</res-auth>
     </resource-ref>
+
+    <resource-ref>
+        <res-ref-name>jdbc/harvest</res-ref-name>
```

Comment:
Not sure about this one - needs a jbndi setup for it.  Is it specific to your setup?  i.e. should it be included?